contrib/docker/build_and_test.sh

```diff
@@ -1,3 +1,17 @@
-#/bin/bash
+#!/bin/bash
 
-docker run --rm -t -i -v $PWD:/io turbodbc_base /io/contrib/docker/build_and_test_in_docker.sh
+# This should run in the 'turbodbc' docker service
+
+set -ex
+
+mkdir -p /turbodbc/build_docker && cd /turbodbc/build_docker
+cmake -DCMAKE_INSTALL_PREFIX=./dist -DPYBIND11_PYTHON_VERSION=3.5 ..
```

Comment:
Use `ninja` for optimal parallelisation based on the host.

```suggestion
cmake -DCMAKE_INSTALL_PREFIX=./dist -DPYBIND11_PYTHON_VERSION=3.5 -GNinja ..
```